apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/DataServerLauncherAppIT.java

```diff
@@ -136,6 +136,871 @@ public void checkAccessLogFormat() {
+            .append(
+                "\"http_access.request.header.IncludeAddressFields\":\"true\",\"http_access.request.header.Accept-Encoding\":\"gzip\"},")
+            .append(
+                "\"logger\":\"HTTP_ACCESS\",\"message\":\"responsecomplete\",\"context\":\"default\"}")
+            .toString();
+
+    String readCoveragePattern = createBeneIdMdcPattern("Coverage", "id", true);
+    assertTrue(Pattern.compile(readCoveragePattern).matcher(coverageV1Read).matches());
+    assertTrue(Pattern.compile(readCoveragePattern).matcher(coverageV2Read).matches());
+    assertFalse(Pattern.compile(readCoveragePattern).matcher(coverageV3Read).matches());
+
+    String readEobPattern = createBeneIdMdcPattern("ExplanationOfBenefit", "id", false);
+    assertTrue(Pattern.compile(readEobPattern).matcher(eobV1Read).matches());
+    assertTrue(Pattern.compile(readEobPattern).matcher(eobV2Read).matches());
+
+    String readPatientPattern = createBeneIdMdcPattern("Patient", "id", true);
+    assertTrue(Pattern.compile(readPatientPattern).matcher(patientV1Read).matches());
+    assertTrue(Pattern.compile(readPatientPattern).matcher(patientV2Read).matches());
+
+    String searchCoveragePattern = createBeneIdMdcPattern("Coverage", "beneficiary", true);
+    assertTrue(Pattern.compile(searchCoveragePattern).matcher(coverageV1Search).matches());
+    assertTrue(Pattern.compile(searchCoveragePattern).matcher(coverageV2Search).matches());
+
+    String searchEobPattern = createBeneIdMdcPattern("ExplanationOfBenefit", "patient", false);
+    assertTrue(Pattern.compile(searchEobPattern).matcher(eobV1Search).matches());
+    assertTrue(Pattern.compile(searchEobPattern).matcher(eobV2Search).matches());
+    assertFalse(Pattern.compile(readCoveragePattern).matcher(eobV3Search).matches());
+
+    String searchPatientByIdPattern = createBeneIdMdcPattern("Patient", "id", false);
+    assertTrue(Pattern.compile(searchPatientByIdPattern).matcher(patientV1SearchById).matches());
+    assertTrue(Pattern.compile(searchPatientByIdPattern).matcher(patientV2SearchById).matches());
+
+    String searchPatientByIdentifierPattern =
+        createBeneIdMdcPattern("Patient", "identifier", false);
+    assertTrue(
+        Pattern.compile(searchPatientByIdentifierPattern)
+            .matcher(patientV1SearchByIdentifier)
+            .matches());
+    assertTrue(
+        Pattern.compile(searchPatientByIdentifierPattern)
+            .matcher(patientV2SearchByIdentifier)
+            .matches());
+
+    String searchPatientByCoverageContract =
+        createBeneIdMdcPattern("Patient", "coverageContractForYearMonth", true);
+    assertTrue(
+        Pattern.compile(searchPatientByCoverageContract)
+            .matcher(patientV1SearchByCoverageContract)
+            .matches());
+    assertTrue(
+        Pattern.compile(searchPatientByCoverageContract)
+            .matcher(patientV2SearchByCoverageContract)
+            .matches());
+  }
```

Comment:
rather than validating hardcoded responses, I was thinking that we would check the MDC and/or the access.json as part of the unit (preferably) or IT tests that actually run the requests. Is that doable?

Reply:
Scrapped these tests and now use the ITs to pull from mdc directly